job-shop-scheduling/job_shop_scheduler.py

```diff
@@ -67,7 +68,9 @@ def get_jss_bqm(job_dict, max_time=None):
           - Hence, at time 0, Job a's 1st task is not run
     """
     scheduler = JobShopScheduler(job_dict, max_time)
-    return scheduler.get_bqm()
+    if stitch_kwargs == None:
+        stitch_kwargs = {}
```

Comment:
Since you're overwriting an argument (in this case, a default argument), I would recommend putting this code directly underneath the doc-string. That way, it's clear to future devs which arguments passed in this function will be overwritten.

For example, imagine if this function was 100 lines long. You don't want someone to have to read 80 lines of code before they find that the `stitch_kwargs` value gets overwritten. It can be pretty shocking to have an argument change values part way through the code.

If it's overwritten early, then anyone wanting to add to this function is able to use `stitch_kwargs`, and can benefit from your overwrite (i.e. `stitch_kwargs = {}`).

Reply:
Done